# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1522093939 25200
#      Mon Mar 26 12:52:19 2018 -0700
# Branch stable
# Node ID 7feafd20703fab1053f92007dad5c93aefd87f71
# Parent  4a6284b539337ef3428292c538a3e6127e38447f
doc: fix doc string declarations (bug #53416)

* graphics.cc (F__image_pixel_size__): Include missing assignment operator in
function type declaration in doc string.
* symtab.cc (F__current_scope__): Likewise.
* audiodevinfo.cc (F__recorder_getaudiodata__): Likewise.

diff --git a/libinterp/corefcn/graphics.cc b/libinterp/corefcn/graphics.cc
--- a/libinterp/corefcn/graphics.cc
+++ b/libinterp/corefcn/graphics.cc
@@ -11540,17 +11540,17 @@ Undocumented internal function.
         gh_manager::execute_callback (h, name, args(2));
     }
 
   return ovl ();
 }
 
 DEFUN (__image_pixel_size__, args, ,
        doc: /* -*- texinfo -*-
-@deftypefn {} {@var{px}, @var{py}} __image_pixel_size__ (@var{h})
+@deftypefn {} {@var{sz} =} __image_pixel_size__ (@var{h})
 Internal function: returns the pixel size of the image in normalized units.
 @end deftypefn */)
 {
   if (args.length () != 1)
     print_usage ();
 
   double h = args(0).xdouble_value ("__image_pixel_size__: argument is not a handle");
 
diff --git a/libinterp/corefcn/symtab.cc b/libinterp/corefcn/symtab.cc
--- a/libinterp/corefcn/symtab.cc
+++ b/libinterp/corefcn/symtab.cc
@@ -635,17 +635,17 @@ determine whether functions defined in f
 ## Test input validation
 %!error (ignore_function_time_stamp ("all", "all"))
 %!error (ignore_function_time_stamp ("UNKNOWN_VALUE"))
 %!error (ignore_function_time_stamp (42))
 */
 
 DEFMETHOD (__current_scope__, interp, , ,
            doc: /* -*- texinfo -*-
-@deftypefn {} {[@var{scope}, @var{context}]} __current_scope__ ()
+@deftypefn {} {[@var{scope}, @var{context}] =} __current_scope__ ()
 Return the current scope and context as integers.
 @seealso{__dump_symtab_info__}
 @end deftypefn */)
 {
   octave::symbol_table& symtab = interp.get_symbol_table ();
 
   octave::symbol_scope scope = symtab.current_scope ();
 
diff --git a/libinterp/dldfcn/audiodevinfo.cc b/libinterp/dldfcn/audiodevinfo.cc
--- a/libinterp/dldfcn/audiodevinfo.cc
+++ b/libinterp/dldfcn/audiodevinfo.cc
@@ -1876,17 +1876,17 @@ get_recorder (const octave_value& ov)
 
   return rec;
 }
 
 #endif
 
 DEFUN_DLD (__recorder_getaudiodata__, args, ,
            doc: /* -*- texinfo -*-
-@deftypefn {} {@var{data}} __recorder_getaudiodata__ (@var{recorder})
+@deftypefn {} {@var{data} =} __recorder_getaudiodata__ (@var{recorder})
 Undocumented internal function.
 @end deftypefn */)
 {
   octave_value retval;
 #if ! defined (HAVE_PORTAUDIO)
   octave_unused_parameter (args);
 
   err_disabled_feature ("__recorder_getaudiodata__",
